File: Iniciante/1064.js
lines = lines.map((e) => Number(e));

let contPos = 0, soma = 0, media;

for (let i = 0; i < lines.length; i++) {
  if (lines[i] > 0) {
    contPos++;
    soma += lines[i];
  }
}

media = soma/contPos;

console.log(`${contPos} valores positivos\n${media.toFixed(1)}`);

File: Iniciante/1064.py
contPos = 0
soma = 0

for i in range(6):
    valor = float(input())
    if valor > 0:
        contPos += 1
        soma += valor
        
media = soma/contPos

print("{} valores positivos".format(contPos))
print("{:.1f}".format(media))
